duckduckgo/danger-settings
# Add alert for trackerblocker URLs being out of sync

Task/Issue URL: https://app.asana.com/0/1200194497630846/1205584545360104/f
Tech Design URL:
CC:

Description:
Add an alert for when TrackerBlocker URLs are updated in a way that they go out of sync between code and script. 

How to test:

- Make sure tests pass. 

- Check iOS implementation running Danger against https://github.com/duckduckgo/iOS/pull/2074: `yarn danger pr https://github.com/duckduckgo/iOS/pull/2074 --dangerfile ./org/allPRs.ts`

- If you have a DDG approved personal token, test macOS implementation running Danger against https://github.com/duckduckgo/macos-browser/pull/1731: `export DANGER_GITHUB_API_TOKEN='<your token>' &&yarn danger pr https://github.com/duckduckgo/iOS/pull/2074 --dangerfile ./org/allPRs.ts`
